README.md

```diff
@@ -957,6 +957,24 @@ features: {
 
 - **features.hideSignOutLinkInMFA** - Hides the sign out link for MFA challenge. Defaults to `false`.
 
+- **features.addButton** - Adds a custom Auth button configurable by the properties below:
+
+    - `text` - String to set button text
```

Comment:
Added a screenshot below. This is only for the Primary Auth page. I don't think we can enable this for any other page.